Makefile

```diff
@@ -150,6 +149,10 @@ generate-latest-dev-csv: operator-sdk dist-csv-generator
 generate-docs: dist-docs-generator
 	hack/docs-generate.sh
 
+.PHONY: generate-manifests-tree
+generate-manifests-tree:
+	hack/generate-manifests-tree.sh
```

Comment:
I see this takes one argument. Where is it used when not here?

Reply:
It is no used. This code wants to do what the old code did, which allowed such optional argument which was never actually used in CI as far as I can tell - corrections welcome!

Reply:
good catch. A variable should have been passed from the Makefile, like the old flow did. Fixing.